Solar-Wrapper/src/TrackablesSolARImpl.cpp

```diff
@@ -43,8 +44,7 @@ namespace org::openapitools::server::implem
 
         //localCRS
         std::vector<float> vector = trackable.getLocalCRS();
-        float* array = &vector[0];
-        Eigen::Matrix4f matrix = Eigen::Map<Eigen::Matrix4f>(array);
+        Eigen::Matrix4f matrix = Eigen::Map<Eigen::Matrix<float,4,4,Eigen::RowMajor>>(vector.data());
```

Comment:
Question: here we can not use `Eigen::Map<Eigen::Matrix4f>` ? Because it `Matrix4f` is column major (see [typedef](https://eigen.tuxfamily.org/dox/group__matrixtypedefs.html#ga3a5de8dfef28d29aed525611e15a37e3) and [default alignement](https://eigen.tuxfamily.org/dox/classEigen_1_1Matrix.html))?

But then, is there no issue to assign this to a `Matrix4f`?